user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -212,10 +209,10 @@ export default function CaseDetailScreen(props: CaseDetailProps) {
         const response = data as Chapter15CaseDetailsResponseData;
         setCaseBasicInfo(response.body?.caseDetails);
       })
-      .then(() => {
-        AttorneysApi.getAttorneys().then((attorneys) => {
-          setAttorneysList(attorneys);
-        });
+      .catch((_error) => {
+        // TODO: handle this error. How to we propogate to the global alert??
```

Comment:
Is this to do now?

Reply:
I think this shows up in multiple files doesn't it?  I remember seeing several comments on this already.